TWUITests/APIStubs/HTTPDynamicStubs.swift

```diff
@@ -17,35 +17,43 @@ import Swifter
 
 protocol HTTPDynamicStubing {
     func update(with stubInfo: APIStubInfo)
+    @available(*, deprecated, message: "Use throwable startServer() instead")
     func start() -> UInt16
+    func startServer() throws -> UInt16
```

Comment:
It's a shame about breaking compatibility... I guess we'll rename `startServer` to `start` when we obsolete the non-throwable one in a major version bump?

Reply:
That was my intention as I did not find a nice way how to not introduce breaking changes otherwise :(

`rename in a major version bump` 👍